build_deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Export vars for helper scripts to use
+# --------------------------------------------
+# name of app-sre "application" folder this component lives in; needs to match for quay
+export COMPONENT="ocp-advisor"
+# IMAGE should match the quay repo set by app.yaml in app-interface
+export IMAGE="quay.io/cloudservices/ocp-advisor-frontend"
+export WORKSPACE=${WORKSPACE:-$APP_ROOT} # if running in jenkins, use the build's workspace
+export APP_ROOT=$(pwd)
```

Comment:
I now it is what is on the docs but sound a bit weird to define a variable after using it as fallback for `WORKSPACE`. Can we double check with other apps?

Reply:
this is indeed strange. in other apps we have the same order: https://github.com/RedHatInsights/insights-inventory-frontend/blob/master/build_deploy.sh, https://github.com/RedHatInsights/vulnerability-ui/blob/master/build_deploy.sh for instance

Reply:
Yes, well, maybe it is not used if `WORKSPACE` is always defined...